app/service/interfaces/i_object_svc.py

```diff
@@ -0,0 +1,25 @@
+import abc
+
+
+class ObjectServiceInterface(abc.ABC):
+
+    @staticmethod
+    @abc.abstractmethod
+    def destroy():
+        """
+        Clear out all data
+        :return:
+        """
+        pass
+
+    @abc.abstractmethod
+    def save_state(self):
+        """
+        Accept all components of an agent profile and save a new agent or register an updated heartbeat.
```

Comment:
is the docstring still accurate?

Reply:
Nope, and I'm quite confused as to how that got there. I'll patch it real quick.